packages/storybook/stories/checkbox.mdx

```diff
@@ -0,0 +1,46 @@
+import { Checkbox } from '@equisoft/design-elements-react';
+import { ArgTypes, Canvas, Meta, Stories, Story } from '@storybook/blocks';
+import * as CheckboxStories  from './checkbox.stories';
+
+
+# Checkbox
+
+## Table of Contents
+    1. [Definition](#definition)
+    2. [Usage](#usage)
+    3. [Variants](#variants)
+    4. [Properties](#properties)
+
+<Meta of={CheckboxStories} />
+
+## Definition
+    Checkboxes allow users to select any number of choices, including zero, one, or several [2].
+    <Canvas of={CheckboxStories.Default} />
+
+## Usage
+    ### When to use
+        - Use when users have to choose between 0 to 6 options.
```

Comment:
Est-ce que nous devrions avoir un "_When not to use_" qui fait mention de ne pas utiliser le composant si nous avons plus de 6 options et d'utiliser le composant dropdown list/combobox variant multiselect?

Reply:
On a des uses cases où est-ce qu'on l'utilise dans le cpanel2... C'est pour ça que je ne l'ai pas mis.